src/cbc_sdk/platform/alerts.py

```diff
@@ -526,6 +526,15 @@ def close(self, closure_reason=None, determination=None, note=None):
             "FALSE_POSITIVE", or "NONE"
             note (str): The comment to set for the alert.
 
+        Note:
+            - This is an asynchronus call that returns a python future.
```

Comment:
This call does not return a Future it returns a job only job.await_completion returns a Future